src/main/java/ti4/map/UnitHolder.java

```diff
@@ -176,6 +179,7 @@ public boolean hasUnits() {
         return !getUnits().isEmpty();
     }
 
+    @JsonProperty("unitsDamage")
     public HashMap<UnitKey, Integer> getUnitDamage() {
         return unitsDamage;
     }
```

Comment:
Made this plural as I think that is more accurate and we will probably want to refactor the function name eventually.